Doc/library/ast.rst

```diff
@@ -2442,9 +2364,42 @@ and classes for traversing abstract syntax trees:
    indents that many spaces per level.  If *indent* is a string (such as ``"\t"``),
    that string is used to indent each level.
 
+   If *show_empty* is ``False``, then empty lists and fields that are ``None``
+   will be omitted from the output for better readability.
```

Comment:
Sorry I didn't notice this the first time, but two nits here:

1) I think we can omit "for better readability" -- it's kind of obvious, and the job of the docs here is to explain what happens, not opine on why it's a good behavior.

2) This sentence reads a bit as if the default is `True`. It should either clarify that `False` is the default, or read the other way around (explain what happens if the flag is `True`).

```suggestion
   If *show_empty* is ``False`` (the default), empty lists and fields that are ``None``
   will be omitted from the output.
```